<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>588</width>
    <height>268</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QPushButton" name="convertBtn">
      <property name="text">
       <string>Convert</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="loadBtn">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Load</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QSpinBox" name="dbIndex">
      <property name="maximum">
       <number>59999</number>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>113</width>
        <height>23</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="4">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="testBtn">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Test</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="clearBtn">
        <property name="text">
         <string>Clear</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="DigitArea" name="digitArea" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>196</width>
        <height>196</height>
       </size>
      </property>
      <property name="toolTip">
       <string>Try to write some digit
here with your mouse</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2" colspan="2">
     <widget class="DigitArea" name="scaledDigitArea" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>196</width>
        <height>196</height>
       </size>
      </property>
      <property name="toolTip">
       <string>How the up scaled input
to the network looks like</string>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="ResultsWidget" name="results" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>588</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExportImg"/>
    <addaction name="actionImportImg"/>
    <addaction name="actionOpenDB"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionExportImg">
   <property name="text">
    <string>Export img</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionOpenDB">
   <property name="text">
    <string>Open DB</string>
   </property>
  </action>
  <action name="actionImportImg">
   <property name="text">
    <string>Import img</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>DigitArea</class>
   <extends>QWidget</extends>
   <header>digit_area.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ResultsWidget</class>
   <extends>QWidget</extends>
   <header>results_widget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
